### ISSUE TITLE
Themes: add tags for Store, Shop, and eCommerce in Theme Showcase

### ISSUE THREAD

**Issue by lancewillett** [Automattic/wp-calypso#39811]
From flow sharing session by our Marketing team (internal reference: pbvYe0-1L-p2)

> We noticed we don’t have a 1) clear e-commerce theme and 2) search functionality is hard to find/use for themes 3) no search results for e-commerce, shop, or related terms appear

An immediate step would be to update the Theme Showcase with the search terms, and point to a few existing themes that work great (even if not “branded” eCommerce, store, or shop).

0. Add "Store" as one of the theme subjects in the Theme Showcase
1. Taxonomy adjustments to any current themes that work well with stores
2. Test in the Theme Showcase

Steps to repeat:

1. Go to https://wordpress.com/themes
2. Scroll down, click "Show All Themes"
3. Search for "store"

Notice that nothing matches.

<img width="989" alt="Screen Shot 2020-03-02 at 11 37 50" src="https://user-images.githubusercontent.com/66797/75706348-4153d500-5c7a-11ea-86be-7a3fb5a5485b.png">

**Comment by davipontesblog** [comment]
Moved to DotCom Manage for triaging.

**Comment by davemart-in** [comment]
@lancewillett do you know who would be the best person to help identify a list of themes that work well for eCommerce? Once I figure out this data, I'm happy to triage this.

**Comment by lancewillett** [comment]
Yes: start with the View team (Ian Stewart, Takashi Irie), then get input maybe from Themes lead Kjell, and Jeffrey Pearce.

**Comment by davemart-in** [comment]
Thanks Lance. 

@ianstewart, @iamtakashi, @kjellr, @jeffikus any recommendations on the best eCommerce themes that are available on WP.com? Once we have a list, I'll triage this to the engineer on our new Janitor rotation.

**Comment by jeffikus** [comment]
@davemart-in sure, there is a "store" tag here -> https://wordpress.com/themes/filter/store for all the themes that have basic WC compatibility. We prefer to recommend the Varia based child themes, child themes listed here -> https://wordpress.com/theme/varia

If you're on a Business plan, you can upload any WC compatible theme.

**Comment by davemart-in** [comment]
Awesome. Thanks @jeffikus!

**Comment by cathymcbride** [comment]
Closing. Please reopen if you think it's needs to be.

**Comment by ianstewart** [comment]
Re-opening this one as I think it's still valuable for WPcom users. It sounds like we have what we need from Themes to make this happen as well.

@simison I'm going to add this one to the Create backlog as we've chatted about this "user setup" kind of work fitting in there. And we can review with @jeffikus when we have a PR up.

**Comment by lancewillett** [comment]
Closing as this is working as expected now.

A search for `store` now shows results.



And, there is now a `Store` top-level filter: https://wordpress.com/themes/filter/store?s=store
